data/json/furniture.json

```diff
@@ -1563,6 +1618,7 @@
         "id" : "f_safe_l",
         "name": "safe",
         "symbol": "X",
+        "description": "What's they keeping? Are they keeping anything? LET'S FIND OUT!",
```

Comment:
Couple things: 

`What could be inside` vs `What's they keeping`. Or, perhaps even better, `What could need this amount of protection?` to differentiate it from, say the open crate text.

Also, there's no "us" in the description. `GO FIND OUT!` instead of `LET'S`

Finally, no need the caps.

All told, I'd recommend: `What could need this amount of protection? Go find out!`